extensions/rest-client/runtime/src/main/java/io/quarkus/restclient/runtime/RestClientRecorder.java

```diff
@@ -60,14 +60,6 @@ public InjectorFactory getInjectorFactory() {
         };
 
         registerProviders(clientProviderFactory, useBuiltIn, providersToRegister, contributedProviders);
-
```

Comment:
I think this is missing the `ResteasyClientBuilderImpl.setProviderFactory(clientProviderFactory)`, similar to the MP REST Client setting

Reply:
@kenfinnigan, I'm not following. Line 63 is
```
RestClientBuilderImpl.setProviderFactory(clientProviderFactory);
```

Reply:
Right, which is the MP REST Client builder impl.

We need to do the same for the JAX-RS Client builder impl, which is `ResteasyClientBuilderImpl`.

The names are close, which makes it confusing

Reply:
D'oh!